### ISSUE TITLE
Extract out edit controller data from the form partial 

### ISSUE THREAD

**Issue by SidharthBansal** [publiclab/plots2#4269]
Due to the signup and login modal integrations, I need to divide the users/_form.html.erb into two parts form2 and form partials. After #4209 is merged. We need to
- [x] Extract out edit controller data from the form partial 
- [ ] Change _form partial to some other name
- [ ] Similarly change name for _form2 partial.

**Comment by kunalvaswani123** [comment]
Hey, I'd like to work on this issue

**Comment by SidharthBansal** [comment]
Hi Kunal. Nice to meet you.
First part of the issue is completed. You can work on the second and third part.
Please ensure that wherever we are using these partials teh name gets changed there.
Please take second subsissue first. After its completion we will do third part in separate PR.

**Comment by kunalvaswani123** [comment]
ok
Could you confirm the changed names?

**Comment by SidharthBansal** [comment]
_form to _edit_form
and _form2 to _create_form

**Comment by kunalvaswani123** [comment]
ok

**Comment by SidharthBansal** [comment]
Please ensure that these changes are present in each and every location from the places from which they were called.
Thanks
